### ISSUE TITLE
Cursor not shown after running all tasks.

### ISSUE THREAD

**Issue by dev-marc-laurent** [SamVerschueren/listr#6]
I'm using listr with [vorpal](https://github.com/dthree/vorpal/) to make a CLI app, and after running the tasks, the cursor is not being redrawn.

I checked the `CLIRenderer` that's based on `log-update` and i realized that function `done` is not being called after on `end` function.

Maybe the idea is to change the render fn, adding a done condition:

```javascript
const render = (tasks, done) => {
	const output = tasks.map(task => task.render());
	logUpdate(output.join('\n'));
	if (done) {
		logUpdate.done();
	}
};
```

**Comment by SamVerschueren** [comment]
Fixed in `0.4.3`. Thanks for reporting :)!

**Comment by dev-marc-laurent** [comment]
Thanks for quick feedback!
